<h2>Why is this an issue?</h2>
<p>This rule checks that the specified child tag does not appear as a direct child of the specified parent.</p>
<h3>Noncompliant code example</h3>
<p>Assuming a parent/child combination of <code>head</code>/<code>body</code>:</p>
<pre>
&lt;head&gt;
  ...
  &lt;body&gt;  &lt;!-- Noncompliant --&gt;
    ...
  &lt;/body&gt;
&lt;/head&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;head&gt;
  ...
&lt;/head&gt;
</pre>

