<h2>Why is this an issue?</h2>
<p>This rule checks that the specified child elements are present inside the specified parent elements.</p>
<h3>Noncompliant code example</h3>
<p>Given a parent/child combination of <code>&lt;head&gt;</code>/<code>&lt;title&gt;</code>:</p>
<pre>
&lt;html&gt;
  &lt;head&gt;
  &lt;/head&gt;  &lt;!-- Noncompliant; no title element --&gt;
  &lt;body&gt;
    ...
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;html&gt;
  &lt;head&gt;
    &lt;title&gt;My Page&lt;/title&gt;
  &lt;/head&gt;
  &lt;body&gt;
    ...
</pre>

