<h2>Why is this an issue?</h2>
<p>Checker to find use of single quote where double quote is preferred.</p>
<h3>Noncompliant code example</h3>
<pre>
&lt;div id='header'&gt;&lt;/div&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;div id="header"&gt;&lt;/div&gt;
</pre>

