<p>Each source file should start with a header stating file ownership and the license which must be used to distribute the application. </p>
<p>This rule must be fed with the header text that is expected at the beginning of every file.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;html&gt;                                    &lt;!-- Noncompliant --&gt;
...
&lt;/html&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;!-- Copyright 2013 SonarSource SA --&gt;
&lt;html&gt;
...
&lt;/html&gt;
</pre>

