<h2>Why is this an issue?</h2>
<p>This rule checks that the specified attributes are not present in HTML tags.</p>
<h3>Noncompliant code example</h3>
<p>For a disallowed attribute list of: <code>a.name</code>:</p>
<pre>
&lt;a href="blah.com" name="Blah link"&gt; &lt;!-- Noncompliant; name attribute is used --&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;a href="blah.com"&gt;
</pre>

