<h2>Why is this an issue?</h2>
<p>This rule checks that the specified HTML elements are not present.</p>
<h3>Noncompliant code example</h3>
<p>For a disallowed list of "font,center":</p>
<pre>
&lt;center&gt;&lt;font color="red"&gt;Hello World!&lt;/font&gt;&lt;/center&gt; &lt;!-- Noncompliant --&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;div class="centerRed"&gt;Hello World!&lt;/div&gt;
</pre>

