<h2>Why is this an issue?</h2>
<p>Whenever a user clicks on a link that targets an image, the website’s navigation menu will be lost.</p>
<p>From a user point of view, it is as if she left the website.</p>
<p>The only way to return to it is using the browser’s 'Back' button.</p>
<p>Instead, it is better to create a page which will display the image using the <code>&lt;img&gt;</code> tag and preserve the navigation menu.</p>
<p>Further, in terms of accessibility, when the image is embedded into a page, content providers are able to provide an alternate text equivalent
through the <code>alt</code> attribute.</p>
<h3>Noncompliant code example</h3>
<pre>
&lt;a href="image.png"&gt;...&lt;/a&gt;  &lt;!-- Non-Compliant --&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;a href="page.html"&gt;...&lt;/a&gt;  &lt;!-- Compliant --&gt;
</pre>

