<h2>Why is this an issue?</h2>
<p>Long pieces of JavaScript should be located in dedicated *.js source files. This makes maintenance of both the script and the pages that use it
easier. Additionally, it offers some efficiencies in serving the files, since it takes better advantage of browser caching to only re-serve the parts
of a web page that have actually changed.</p>
<h3>Noncompliant code example</h3>
<p>With the default parameter value of 5:</p>
<pre>
&lt;head&gt;
  ...
  &lt;script type="text/javascript" language="JavaScript"&gt;  &lt;!-- Noncompliant --&gt;
    function doTheThing(arg1) {
      ...
      ...
    }

    function doTheOtherThing(arg1) {
      ...
    }

    function andSoOn() {
      ...
    }
  &lt;/script&gt;
&lt;/head&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;head&gt;
  ...
  &lt;script type="text/javascript" language="JavaScript" src="myLongScript.js"&gt; &lt;/script&gt;
&lt;/head&gt;
</pre>

