<h2>Why is this an issue?</h2>
<p>Favicons are shown for example in the browserâ€™s address bar, bookmark list, or tabs.</p>
<p>They enable users to quickly identify and recognize websites.</p>
<h3>Noncompliant code example</h3>
<pre>
&lt;head&gt;                                                                  &lt;!-- Noncompliant --&gt;
  &lt;title&gt;...&lt;title&gt;
&lt;/head&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;head&gt;
  &lt;title&gt;...&lt;title&gt;
  &lt;link rel="shortcut icon" href="http://example.com/myicon.ico" /&gt;
&lt;/head&gt;
</pre>

