<h2>Why is this an issue?</h2>
<p>Titles are important because they are displayed in search engine results as well as the browserâ€™s toolbar.</p>
<p>This rule verifies that the <code>&lt;head&gt;</code> tag contains a <code>&lt;title&gt;</code> one, and the <code>&lt;html&gt;</code> tag a
<code>&lt;head&gt;</code> one.</p>
<h3>Noncompliant code example</h3>
<pre>
&lt;html&gt;         &lt;!-- Non-Compliant --&gt;

&lt;body&gt;
...
&lt;/body&gt;

&lt;/html&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;html&gt;         &lt;!-- Compliant --&gt;

&lt;head&gt;
&lt;title&gt;Some relevant title&lt;/title&gt;
&lt;/head&gt;

&lt;body&gt;
...
&lt;/body&gt;

&lt;/html&gt;
</pre>

