<h2>Why is this an issue?</h2>
<p>It is considered best-practice to use relative URLs in web pages to prevent having to update the addresses if the web address in use changes.
Moreover, if some absolute URLs are missed in such a process, it will obviously impact the user experience.</p>
<h3>Noncompliant code example</h3>
<pre>
&lt;img src="http://www.myserver.com/smiley.gif" alt="Smiley face" height="42" width="42" /&gt;
</pre>
<h3>Compliant solution</h3>
<pre>
&lt;img src="smiley.gif" alt="Smiley face" height="42" width="42" /&gt;
</pre>

