<p>In order to be accessible to visually impaired users, it is important that tables have a caption briefly describing its contents.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;table&gt;                                                         &lt;!-- Noncompliant --&gt;
  ...
&lt;table&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;table&gt;
  &lt;caption&gt;New York City Marathon Results 2013&lt;/caption&gt;
  ...
&lt;table&gt;
</pre>

